stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/view/MessageListView.kt

```diff
@@ -152,14 +154,14 @@ public class MessageListView : ConstraintLayout, IMessageListView {
         MessageLongClickListenerView { _, view ->
             optionsView = view
 
-            binding.blurLayer.isVisible = true
+            binding.shadowLayer.isVisible = true
 
             if (view.parent != null && view.parent is ViewGroup) {
                 (view.parent as ViewGroup).removeView(view)
```

Comment:
Would be cleaner to refactor later and recreate the message view holder with viewholder factory. Would probably resolve the issue with message displayed without avatar (in the message list only last message in a row is displayed with avatar) and the issue with the list adjusting its scroll position.

Reply:
Checked the design, looks like avatar should not be displayed at all

Reply:
My bad, checked the design, looks like a message should be displayed just like in the message list (either with avatar or without it)